README.md

```diff
@@ -61,7 +61,7 @@ VM settings within the `Vagrantfile` once it has been symlinked.
 8. After step 6, you'll be in the container and then can compile and test NetBricks via
```

Comment:
Note: I will tackle a more complete README as well (and link to old netbricks version) in my next PR with more tests.